test/runtests.jl

```diff
@@ -176,7 +176,7 @@ end
 
 @testset "Gamma IdentityLink" begin
     gm10 = fit(GeneralizedLinearModel, lot1 ~ 1 + u, clotting, Gamma(), IdentityLink(),
-        convTol=1e-8)
```

Comment:
Here: https://github.com/JuliaStats/GLM.jl/pull/182#pullrequestreview-39608592. I think what happened is that mis-interpreted a comment by @ararslan.